src/porting/jsonPorter.hh

```diff
@@ -74,8 +74,13 @@ void simulationFromJSON(std::string jsonFile, arch::Network<T>* network_, sim::S
         readDropletInjections<T>(jsonString, simulation, activeFixture);
     } else
     if (platform == sim::Platform::Mixing) {
-        // NOT YET SUPPORTED
-        throw std::invalid_argument("Mixing simulations are not yet supported in the simulator.");
+        if (simType != sim::Type::Abstract) {
+            throw std::invalid_argument("Mixing simulations are currently only supported for 1D simulations.");
```

Comment:
for Abstract simulations

Reply:
Fixed with [0350b37](https://github.com/cda-tum/mmft-simulator/pull/16/commits/0350b37ec135c321cd1f0a480a5d6e275a668c52)